Code/Engine/Input/InputOutputUtils.cpp

```diff
@@ -597,3 +597,14 @@ std::string GetFileDirectory(const std::string& filePath)
 
     return "ERROR";
 }
+
+//-----------------------------------------------------------------------------------
+bool IsDirectory(const std::wstring& path)
+{
+    DWORD fileAttribute = GetFileAttributes(path.c_str());
+    if (fileAttribute == FILE_ATTRIBUTE_DIRECTORY)
```

Comment:
Is the file attribute a single value, or is it a bitfield? Meaning, does Windows fit multiple values into this DWORD (such as a directory bit, a is readable bit, etc.), or is it safe to make this equals comparison because they're unique values?